### PR TITLE
Don't change the default session_handler if it is allready set Ref: #5127

### PR DESCRIPTION
**Pull request by zero-24**
#### Steps to reproduce the issue

We are using memcache as default session handler and this set up is in php.ini.
#### Expected result

Joomla to use default session handler is not set in configuration.
#### Actual result

PHP Warning:  session_start(): open(tcp://127.0.0.1:11211/sess_r5c5940ecaru55pr69bssctmu2, O_RDWR) failed: No such file or directory
#### System information (as much as possible)

php.ini: session.save_handler = memcache
#### Additional comments

Change libraries/joomla/session/storage/none.php to use default php session handler.

See: #5127 by @Olvikolvi

### CODE REVIEW

**Comment by Fedik** [comment]
found old one #2695, that can be closed after current will be merged

File: libraries/joomla/session/storage/none.php [hunk 1]
```diff
@@ -26,6 +26,12 @@ class JSessionStorageNone extends JSessionStorage
 	 */
 	public function register()
 	{
-		ini_set('session.save_handler', 'files');
+		$handler = ini_get('session.save_handler');
+
+		// Set session.save_handler to files only if nothing else is set
+		if ($handler = '')
```
Comment (Fedik) [inline libraries/joomla/session/storage/none.php]: looks like here the mistake,
by call `if ($handler = '')` you reset `$handler`  variable to empty value, so it always `false`
I think just enough `if(!ini_get('session.save_handler'))`

**Comment by zero-24** [comment]
Thanks @Fedik fixed with: https://github.com/joomla/joomla-cms/commit/7e751e1d2218e74d3d67a3ecd9bd5f6173f84c18

**Comment by Fedik** [comment]
@zero-24  still small mistake ;)

I just thought, maybe also can remove this construction, as [default PHP 'session.save_handler' is 'files'](http://php.net/manual/en/session.configuration.php)

**Comment by zero-24** [comment]
done @Fedik

**Comment by Fedik** [comment]
@zero-24  thanks!

**test**
works good

**Comment by zero-24** [comment]
Based on testing and a other PR that is tested and that whant to add the same code we can move this to RTC.

Thanks for testing here @Fedik and @Olvikolvi !<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5142">issues.joomla.org/joomla-cms/5142</a>.</sub>

**Comment by oparoz** [comment]
:+1:

**Comment by phproberto** [comment]
Merged. Thanks!

**Pull request closed without merging**